examples/backpack-demo/index.js

```diff
@@ -7,22 +7,204 @@
 /**
  * @fileoverview Backpack demo initialization.
  */
+var toolbox = {
```

Comment:
Can you change this to `const` instead of `var`?

Reply:
You are right, `var` should be avoided.